src/atopile/cli/build.py

```diff
@@ -147,4 +147,4 @@ def clone_footprints(build_args: BuildContext) -> None:
 
     for component in all_components:
         log.debug("Cloning footprint for %s", component)
-        download_footprint(component, dir=build_args.build_path / "build/footprints/footprints.pretty")
+        download_footprint(component, footprint_dir=build_args.build_path / "build/footprints/footprints.pretty")
```

Comment:
is build path already in /build? I think this should be "footprints/footprints.pretty"